MapboxMobileEvents/MMEHashProvider.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+#import "MMEEventsConfiguration.h"
+
+@interface MMEHashProvider : NSObject
+
+@property (nonatomic) NSArray *cnHashes;
+@property (nonatomic) NSArray *comHashes;
```

Comment:
Are these only supposed to be set by `updateHashesWithConfiguration `? If so, you might want to declare these as `readonly` in the header file, then redeclare as `readwrite` in the implementation file.

Reply:
Yes! Good call